update-imported-docs/update-imported-docs.py

```diff
@@ -203,7 +205,9 @@ def main():
     # create the temp work_dir
     try:
         print("Making temp work_dir")
-        work_dir = tempfile.mkdtemp()
+        work_dir = tempfile.mkdtemp(
+            dir='/tmp' if platform.system() == 'Darwin' else tempfile.gettempdir()
+        )
```

Comment:
Hello @Colstuwjx .
The value of $TMPDIR, $TEMP, or $TMP does not return as expected on macOS?

Reply:
See the issue [19666 comment](https://github.com/kubernetes/website/issues/19666#issue-582303320), the temp dir is sth like `/var/folders/0g/vc5bvn8x1cq63y3fwt68n64c0000gn/T/tmpe6r4z491/src/...`, and while docker run with `-v` option, it would panic since mount permission denied.